soil-service/app/services/calculate_humus_balance_service.rb

```diff
@@ -0,0 +1,38 @@
+class CalculateHumusBalanceService
+  include Singleton
+
+  CONSECUTIVE_YEAR_CROP_VALUE = 1.3
+
+  def get_farmer_humus_balance(body)
+    return unless body.present?
+    seasonal_crops = JSON.parse(body)
```

Comment:
Parsing of the request body shouldn't belong to this service. Service is ideally dealing with one single thing and arguments passed to service's methods should be already in correct format.